draft-ietf-quic-recovery.md

```diff
@@ -616,6 +553,50 @@ sender might choose to optimize this by setting the timer fewer times if it
 knows that more ack-eliciting packets will be sent within a short period of
 time.
 
+The retransmission timer is not set if the time threshold
+{{time-threshold}} loss detection timer is set.  The time threshold loss
+detection timer is expected to both expire earlier than the PTO and be less
+likely to spuriously retransmit data. The Initial and Handshake packet number
+spaces will typically contain a small number of packets, so losses are less
+likely to be detected using packet-threshold loss detection.
+
+## Handshakes and new paths
+
+The initial probe timeout for a new connection or new path SHOULD be
+set to twice the initial RTT.
+
+Resumed connections over the same network SHOULD use the previous connection's
+final smoothed RTT value as the resumed connection's initial RTT.  If no
+previous RTT is available, or if the network changes, the initial RTT SHOULD
+be set to 500ms, resulting in a 1 second initial timeout as recommended in
+{{?RFC6298}}.
+
+A connection MAY use the delay between sending a PATH_CHALLENGE and receiving
+a PATH_RESPONSE to seed initial_rtt for a new path, but the delay SHOULD NOT
+be considered an RTT sample.
+
+Until the server has validated the client's address on the path, the amount of
+data it can send is limited, as specified in Section 8.1 of {{QUIC-TRANSPORT}}.
+If not all unacknowledged CRYPTO data can be sent, then all unacknowledged
+CRYPTO data sent in Initial packets should be retransmitted.  If no data can be
+sent, then no alarm should be armed until data has been received from the
+client.
```

Comment:
How about idle timeout / connection abort?

Reply:
The idle timeout is a transport concept, it has nothing to do with loss recovery.

Reply:
Maybe say "no recovery alarm" or "no retransmission alarm"